### PR TITLE
Support of ssl and application on same server

### PR DESCRIPTION
**Pull request by yjacolin**
The ssl chapter explain how to configure an application behind a proxy
with https url.

Most use case now, locate the ssl certificat on the same server as the
application, trigger new issue which should be adressed by this
new section in documentation.

### CODE REVIEW

File: doc/integrator/https.rst [hunk 1]
```diff
@@ -8,6 +8,14 @@ generated by the application (CSS and Javascript files, images, MapServer
 requests, etc.) use the HTTPS scheme as well. Otherwise the browser will
 prompt "unsecure content" warnings.
 
+There is two ways to manage this:
+
+* application behind a proxy
+* application and ssl certificat on the same server
```
Comment (sbrunner) [inline doc/integrator/https.rst]: ssl => SSL

File: doc/integrator/https.rst [hunk 2]
```diff
@@ -34,6 +42,24 @@ value of ``request.scheme``. For example::
     /* ... */
     };
 
+Application and ssl certificat on the same server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (sbrunner) [inline doc/integrator/https.rst]: same

Comment (sbrunner) [inline doc/integrator/https.rst]: also bellow ...

File: doc/integrator/https.rst [hunk 3]
```diff
@@ -34,6 +42,24 @@ value of ``request.scheme``. For example::
...
+
+If the ssl certificat and the application are located on the same server, all 
+requests will be redirect to https. So you should change the scheme to https 
+for all url except for some parameters that should always use http (actually 
+all request on localhost): *url* parameter in tilegeneration configuration.
+
+If you apply ssl encryption on your application, you should take care of the 
+tiles url to use https scheme to avoid mixed schemes: change *tiles_url* in
```
Comment (sbrunner) [inline doc/integrator/https.rst]: "mixed schemes" => "secure and insecure contents" ?

File: doc/integrator/https.rst [hunk 4]
```diff
@@ -8,6 +8,14 @@ generated by the application (CSS and Javascript files, images, MapServer
 requests, etc.) use the HTTPS scheme as well. Otherwise the browser will
 prompt "unsecure content" warnings.
 
+There is two ways to manage this:
```
Comment (kalbermattenm) [inline doc/integrator/https.rst]: is -> are

**Comment by yjacolin** [comment]
Don't know why "All checks failed", seems not related to my PR. Isn't it?

**Comment by ochriste** [comment]
doc/integrator/https.rst:48: trailing whitespace.

+If the SSL certificate and the application are located on the same server, all 

doc/integrator/https.rst:49: trailing whitespace.

+requests will be redirect to https. So you should change the scheme to https 

doc/integrator/https.rst:50: trailing whitespace.

+for all url except for some parameters that should always use http (actually 

doc/integrator/https.rst:53: trailing whitespace.

+If you apply ssl encryption on your application, you should take care of the 

doc/integrator/https.rst:54: trailing whitespace.

+tiles url to use https scheme to avoid secure and insecure contents: change 

doc/integrator/https.rst:57: trailing whitespace.

+Finaly, you should redirect all http request to https scheme. On Camptocamp's 

doc/integrator/https.rst:60: trailing whitespace.

+In case you load http external resources into your application, you should use

**Comment by sbrunner** [comment]
And you should squash your commits into one...

**Comment by yjacolin** [comment]
Thanks Oliver.
@sbrunner done, could I merge it?

**Comment by sbrunner** [comment]
Looks good for me :-)

**Pull request merged**